scripts/README.md

```diff
@@ -7,6 +7,10 @@ Below is a list of scripts available, along with a simple description of
 what each one does. The details of what they are doing is available within the
 script.
 
+[`approve-backstop`](approve-backstop)
+Approves changes found during the last run of Backstop stops. Backstop runs in
```

Comment:
Did you mean "Backstop tests", not "Backstop stops"?

Reply:
Indeed I did. Good spot 👀